src/Parser/FrostMage/Modules/Features/FingersFrost.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage, formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Module from 'Parser/Core/Module';
+
+class FingersFrostTracker extends Module {
+
+	overwrittenFingersFrostProcs = 0;
+	totalFingersFrostProcs = 0;
+
+	static dependencies = {
+		combatants: Combatants,
+	}
+
+	on_initialized(){}
```

Comment:
Empty function.

Reply:
Same as the Brain Freeze thing, i didnt know if it was required or not, so i left it blank. ill remove it.